parm/wave/ww3_multi.gefs.inp.tmpl

```diff
@@ -0,0 +1,38 @@
+$ WAVEWATCH III multi-grid input file
```

Comment:
At this point this is not needed for GEFS, so I'm not sure if it's worth creating this file?

Reply:
Maybe not, but even the GFS isn't being used currently.

Is there any reason these files would be different for GFS and GEFS? We could just use symlinks for these two (`ww3_multi` and `ww3_shel`) if not (or just take the `$RUN` out of the filename).

Reply:
Fair enough. 

These should be configurable enough to handle GFS or GEFS and allow for taking $RUN out of the filename.